common/changes/@microsoft/rush/silence-cache-logging-in-quiet-mode_2023-06-06-22-19.json

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
Accidental double change file.

Reply:
Weird, these were supposed to be different.